### PR TITLE
fix(core/webidl): autolink link Window and object

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: src/core/webidl.js [hunk 1]
```diff
@@ -38,8 +38,20 @@ const templates = {
...
   },
-  reference(wrapped) {
-    return hyperHTML`<a data-xref-type="_IDL_">${wrapped}</a>`;
+  reference(wrapped, unescaped) {
+    let type = "_IDL_";
+    let cite = null;
+    switch (unescaped) {
+      case "Window":
+        type = "interface";
+        cite = "HTML";
```
Comment (saschanaz) [inline src/core/webidl.js]: https://respec.org/xref/ only gives a singnle item for `Window`, so not sure why we need this...

Comment (marcoscaceres) [reply src/core/webidl.js]: Ok, let’s remove 👍

Comment (saschanaz) [reply src/core/webidl.js]: Removing it reintroduces the bug. Hard to understand...

Comment (marcoscaceres) [reply src/core/webidl.js]: Magic 👏

Comment (marcoscaceres) [reply src/core/webidl.js]: But seriously, let’s run with this and then deal with xref not matching separately as most specs are  currently in a broken state.

Comment (saschanaz) [reply src/core/webidl.js]: Agreed.

Comment (saschanaz) [reply src/core/webidl.js]: Will you publish this or should I do it instead?

Comment (marcoscaceres) [reply src/core/webidl.js]: Sorry, just saw this... published. Still have the issue with Exposed=Worker... but it only affects a few specs. We can deal with it tomorrow.

**Pull request merged**